### DISCUSSION TITLE
feat: input groups

### DISCUSSION THREAD

**Discussion by verdie-g** [microsoft/fluentui-blazor#1870] [Ideas]
# 🙋 Feature Request

Being able to combine form inputs like that:

![image](https://github.com/microsoft/fluentui-blazor/assets/9092290/4256251f-d149-4f12-bce5-ee9ef762e351)

**Comment by vnbaaij** [comment]
Hi,

This is not something we are planning for at the moment.

Will convert this to a discussion item so others can chime in and let us know if this has broader interest.

**Comment by verdie-g** [comment]
Actually this is not part of fluent ui so I don't think it should be included in this library either.
